cas-server-core/src/main/java/org/jasig/cas/services/AbstractRegisteredService.java

```diff
@@ -82,12 +83,6 @@
     @Column(name = "proxy_policy", nullable = false)
     private RegisteredServiceProxyPolicy proxyPolicy = new RefuseRegisteredServiceProxyPolicy();
 
-    @Column(name = "enabled", nullable = false)
```

Comment:
people are going to have to change their local DB DDL since these were not nullable and now they'll never be set.